packages/mlx/mlx.0.9/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "OCaml syntax dialect which adds JSX syntax expressions"
+maintainer: ["Andrey Popp"]
+authors: ["Andrey Popp"]
+license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
+homepage: "https://github.com/ocaml-mlx/mlx"
+doc: "https://url/to/documentation"
+bug-reports: "https://github.com/ocaml-mlx/mlx/issues"
+depends: [
+  "ocaml" {>= "4.14.0"}
+  "ppxlib"
+  "dune" {>= "3.9"}
```

Comment:
```suggestion
  "dune" {>= "3.15"}
```

Because https://github.com/ocaml-mlx/mlx/blob/f465250b9607e3c78e78153962fe0cb4e05181ac/test/example/dune-project#L4 has `lang dune 3.15`

I think this is the cause for the CI failure.